code/datums/diseases/transformation.dm

```diff
@@ -127,15 +127,18 @@
 	if(affected_mob.mind && !is_monkey(affected_mob.mind))
 		add_monkey(affected_mob.mind)
 	if(ishuman(affected_mob))
-		if(affected_mob && !is_monkey_leader(affected_mob.mind))
-			var/mob/living/carbon/monkey/M = affected_mob.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSE)
+		GLOB.huds[DATA_HUD_MEDICAL_ADVANCED].add_hud_to(affected_mob)
+		if(affected_mob && !is_monkey_leader(affected_mob.mind) && prob(96))
+			var/mob/living/carbon/monkey/M = affected_mob.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPVIRUS | TR_KEEPSE)
 			M.ventcrawler = VENTCRAWLER_ALWAYS
 		else
 			affected_mob.junglegorillize()
 
 /datum/disease/transformation/jungle_fever/stage_act()
 	..()
 	switch(stage)
+		if(1)
+			GLOB.huds[DATA_HUD_MEDICAL_ADVANCED].add_hud_to(affected_mob)
```

Comment:
It shouldn't give them the hud when they have the disease, but when they have the moneyness